frontend/src/ui-components/AcmDropdown/AcmDropdown.tsx

```diff
@@ -61,38 +61,42 @@ export type AcmDropdownItems = {
   component?: React.ReactNode
 }
 
+const getBackgroundColor = (props: AcmDropdownProps): string => {
+  if (props.isDisabled) {
+    return 'var(--pf-v5-global--disabled-color--200)'
+  }
+  if (props.isPrimary) {
+    return 'var(--pf-v5-global--primary-color--100)'
+  }
+  return 'transparent !important'
+}
+
 const useStyles = (props: AcmDropdownProps) => ({
   button: css({
-    '& button': {
-      backgroundColor: props.isDisabled
-        ? 'var(--pf-v5-global--disabled-color--200)'
-        : props.isPrimary
-          ? 'var(--pf-v5-global--primary-color--100)'
-          : 'transparent',
-      '&:hover': {
-        backgroundColor: props.isPrimary
-          ? 'var(--pf-v5-global--primary-color--200)'
-          : 'var(--pf-v5-global--BackgroundColor--200)',
+    '& .pf-v5-c-menu-toggle': {
+      backgroundColor: getBackgroundColor(props),
+
+      '&:hover, &:focus, &:active': {
+        backgroundColor: props.isPrimary ? 'var(--pf-v5-global--primary-color--200)' : 'transparent !important',
+        color: props.isPrimary ? 'var(--pf-v5-global--Color--light-100)' : 'var(--pf-v5-global--Color--100)',
```

Comment:
Could you try just stripping out all of this custom CSS? I'm not sure why this was ever needed. Seems to look better without it.